img2lambda/cli/main.go

```diff
@@ -5,6 +5,7 @@ package main
 import (
 	"errors"
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Instead of importing the SDK here, you can collect the runtimes as []string, then use aws.StringSlice in the conversion method.

Reply:
that's a good idea. i have done that now